src/explore-education-statistics-common/src/components/form/FormFieldCheckboxSearchSubGroups.tsx

```diff
@@ -16,6 +16,7 @@ export type FormFieldCheckboxSearchSubGroupsProps<FormValues> = OmitStrict<
 function FormFieldCheckboxSearchSubGroups<FormValues>(
   props: FormFieldCheckboxSearchSubGroupsProps<FormValues>,
 ) {
+  const { onSubGroupAllChange, onChange } = props;
```

Comment:
Slight nitpick, but would like to have a line break before any `return` to make the formatting a little nicer so that things aren't crowded.

Would do this for the rest of the files in this PR where we've introduced similar changes.